### ISSUE TITLE
Allow to customize font family

### ISSUE THREAD

**Issue by bongnv** [vuepressjs/vuepress-theme-blog#79]
#### What problem does this feature solve?
Currently, font-family in some places is hard-coded to `PT Serif, Serif`. Therefore, if you want to use a different font for your own need, it's not possible unless:
- Override the view
- Override @font-face

I believe both of the approaches are not elegant.

#### What does the proposed API look like?
Allow to override `$font-family-serif` in `index.styl` or `palette.styl` like:
```stylus
$font-family-serif = 'Noto Serif', Serif
```

#### How should this be implemented in your opinion?
Some hard-coded place like:
```stylus
.vuepress-blog-theme-content
  font-family PT Serif, Serif
```
will be changed to 
```stylus
.vuepress-blog-theme-content
  font-family $font-family-serif
```

#### Are you willing to work on this yourself?
Yes

**Comment by newsbielt703** [comment]
Yes, there're still some styles should be able to override in both default theme and blog theme. 

 I think the variable name can be `$fontFamily`. Contribution Welcome!

**Comment by Kocal** [comment]
I am probably missing something, but how can we override [`font.style`](https://github.com/vuepressjs/vuepress-theme-blog/blob/master/styles/font.styl) and prevent font files import?

**Comment by bongnv** [comment]
Raised the PR #80.
